app/views/layouts/application.html.erb

```diff
@@ -8,19 +8,79 @@
+    <link href="../../assets/stylesheets/output.css" rel="stylesheet">
   </head>
 
+  <header class="selection:bg-blue-500 selection:text-white">
+    <nav>
+      <div class="flex flex-row justify-between items-center text-white bg-gray-600 px-[1rem] py-[1rem] sm:px-[1.5rem] sm:py-[1.5rem]">
+        <%= link_to "Root", root_path %>
+        <ul class="fixed hidden md:block right-0 top-0 md:relative md:flex-row md:justify-between" id="navbar">
+          <div class="flex flex-col md:flex-row gap-4 md:gap-0 md:mt-0 px-[1rem] py-[1rem] sm:px-[1.5rem] sm:py-[1.5rem] md:p-0 min-h-[100vh] md:min-h-0 min-w-[40vw] md:min-w-0 bg-gray-600/[0.95] z-1 md:z-0 rounded-lg">
+            <li class="md:hidden text-left cursor-pointer" id="navbar__closeButton">
+              <span class="material-symbols-rounded hover:text-blue-400">
+                close
+              </span>
+            </li>
+            <hr class="md:hidden">
+            <li>
+              <a class="hover:bg-blue-400 hover:text-white p-2 md:p-3 rounded-md hover:underline cursor-pointer" href="#">About</a>
+            </li>
+            <hr class="md:hidden">
+            <li>
+              <a class="hover:bg-blue-400 hover:text-white p-2 md:p-3 rounded-md hover:underline cursor-pointer" href="#">FAQ</a>
+            </li>
+            <hr class="md:hidden">
+            <li>
+              <a class="hover:bg-blue-400 hover:text-white p-2 md:p-3 rounded-md hover:underline cursor-pointer" href="#">Send a request</a>
+            </li>
+            <hr class="md:hidden">
+            <li>
+              <a class="hover:bg-blue-400 hover:text-white p-2 md:p-3 rounded-md hover:underline cursor-pointer" href="#">Contact</a>
+            </li>
+            <hr class="md:hidden">
+            <li>
+              <% if current_user.present? %>
+                <span data-test="current-user-email"><%= current_user.email %></span>
+                <%= button_to "Sign out", destroy_user_session_path, method: :delete, form_class: "" %>
+              <% else %>
+                <%= link_to "Login", new_user_session_path %>
+              <% end %>                              
+            </li>
+            <hr class="md:hidden">
+          </div>
+        </ul>
+        <div class="md:hidden cursor-pointer" id="navbar__menuButton">
+          <span class="material-symbols-rounded">
+            menu
+          </span>
+        </div>
+      </div>			
+    </nav>
+  </header>
+
   <body>
     <p data-test="notice" class="notice"><%= notice %></p>
     <p data-test="alert" class="alert"><%= alert %></p>
-    <%= link_to "Root", root_path %>
-    <% if current_user.present? %>
-      <span data-test="current-user-email"><%= current_user.email %></span>
-      <%= button_to "Sign out", destroy_user_session_path, method: :delete, form_class: "d-inline" %>
-    <% else %>
-      <%= link_to "Login", new_user_session_path %>
```

Comment:
Why those login links are removed ?

Reply:
They're not. They have been moved to navbar.

Reply:
Sorry, my mistake. I saw body `<body>` tag and not login buttons.
Can you move `<header>` to  `<body>` so we have proper html ?

Reply:
Moved to <body>